login/user_claims_provider_test.go

```diff
@@ -0,0 +1,161 @@
+package login
```

Comment:
Ich würde hier keine snabble.io domaine in den testdaten nehmen, sondern example.com